tests/test_xivo_update_odbcinst_config.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2015-2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0+
```

Comment:
FYI: the SPDX-License-Identifier is now [GPL-3.0-or-later](https://spdx.org/licenses/GPL-3.0-or-later.html); `GPL-3.0+` has been [deprecated](https://spdx.org/licenses/GPL-3.0+.html)

Reply:
Ok, sorry, I had just copied the other test. Should I replace the others too? Or not touch them?

Reply:
Feel free to fix them all (in any repo). Just don't touch the copyright date for this change

Reply:
Ok, thanks! I'll keep an eye out.